sequencer/src/bin/keygen.rs

```diff
@@ -0,0 +1,54 @@
+//! Utility program to generate keypairs
+
+use async_compatibility_layer::logging::setup_logging;
+
+use anyhow::{self, bail, Context};
+use clap::Parser;
+use hotshot::types::SignatureKey;
+use hotshot_types::signature_key::BLSPubKey;
+
+const DEFAULT_SEED : &str = "[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]";
+
+/// Utility program to generate keypairs
+#[derive(Clone, Debug, Parser)]
+struct Options {
+    /// Seed for generating keypair.
+    /// Default is all zeros.
+    #[clap(long, short = 's',  default_value = DEFAULT_SEED, value_parser = parse_seed)]
+    seed: [u8; 32],
```

Comment:
IMO the default should be generate a seed from entry (ie if you use the program with no arguments it generates a cryptographically random key).

You can accomplish this using `let mut rng = rand_chacha::ChaChaRng::from_entropy()`, and then `rng.fill_bytes` to populate a 32-byte array with random data for passing to `generated_from_seed_indexed`

Reply:
Oh and if we generate a random seed it would be nice to print it out along with the private key